sites/public/src/ListingView.tsx

```diff
@@ -514,7 +514,12 @@ export const ListingView = (props: ListingProps) => {
             </Heading>
             <p className="text-gray-750 text-base mb-1">{listing.developer}</p>
             <p className="text-base">
-              <a href={googleMapsHref} target="_blank" aria-label="Opens in new window">
+              <a
+                href={googleMapsHref}
+                target="_blank"
+                rel="noreferrer noopener"
+                aria-label={t("t.viewOnMap")}
```

Comment:
I believe this aria-label isn't needed in this case since the link text itself provides the same description.

Reply:
+1 but it seemed like this was the ask from the ac?

Reply:
Yeah I see that as well 🤔 and I think my concerns above on the labeling are more on the AC as well. Wish I had gone down the rabbit hole before grooming but recognizing the state of this work and that the barrier referenced above already existed with our "Opens in new window" override, I could also capture these thoughts in a follow-up issue. Do you think that path would be preferable @emilyjablonski?